babel.config.js

```diff
@@ -304,13 +304,34 @@ module.exports = function (api) {
     config.plugins.push("babel-plugin-istanbul");
   }
 
+  let caller;
+  api.caller(data => {
+    caller = data;
+  });
+
+  if (caller && caller.name == "@babel/eslint-parser") {
+    Object.assign(config, {
```

Comment:
Note: This is risky, because it also disables any syntax plugin. However, we currently only use `@babel/eslint-parser` for standard JS files so that's fine.

```suggestion
    // We disable all the plugins and presets, since we are only interested in parsing JS.
    Object.assign(config, {
```